project_euler/problem_09/sol1.py

```diff
@@ -30,6 +29,38 @@ def solution():
                             return a * b * c
 
 
+def solution_fast():
+    """
+     Returns the product of a,b,c which are Pythagorean Triplet that satisfies
+     the following:
+     1. a < b < c
+     2. a**2 + b**2 = c**2
+     3. a + b + c = 1000
+
+    # The code below has been commented due to slow execution affecting Travis.
+    # >>> solution_fast()
+    # 31875000
+    """
+    for a in range(300):
+        for b in range(400):
+            c = 1000 - a - b
+            if a < b < c and (a ** 2) + (b ** 2) == (c ** 2):
+                return a * b * c
+
+
+def benchmark() -> None:
+    """
+    Benchmark code comparing two different version function.
+    """
+    import timeit
+
+    print(timeit.timeit("solution()", setup="from __main__ import solution", number=1))
+    print(
+        timeit.timeit(
+            "solution_fast()", setup="from __main__ import solution_fast", number=1
```

Comment:
```suggestion
            "solution_fast()", setup="from __main__ import solution_fast", number=1000
```
Just to make sure it is not a fluke.

Reply:
If we replace number `1` with `1000`. It take a long time to complete.